.github/workflows/ci.yml

```diff
@@ -140,13 +140,22 @@ jobs:
           cache: "pip"
 
       - name: Installing Linux Dependencies
-        if: ${{ runner.os == 'Linux' && matrix.os != 'ubuntu-20.04' && !env['TEST_X86'] }}
+        if: ${{ runner.os == 'Linux' && matrix.os != 'ubuntu-22.04' && !env['TEST_X86'] }}
         run: |
           sudo apt update
-          sudo apt install cmake clang-14 clang-tools llvm kcov g++-12 valgrind zlib1g-dev libcurl4-openssl-dev
+          sudo apt install cmake clang-15 clang-tools llvm g++-12 valgrind zlib1g-dev libcurl4-openssl-dev
```

Comment:
Changed to clang-15 (not sure why this was clang-14 before?)

Reply:
Since we switched to `24.04`, you can now use `clang-18` (the default `clang` on that distro version).